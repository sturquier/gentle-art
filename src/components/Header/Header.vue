<template>
  <v-app-bar>
    <router-link to="/">
      <img
        alt="logo"
        class="logo"
        src="@/assets/images/logo.svg"
      />
    </router-link>
  </v-app-bar>
</template>

<style>
  .logo {
    width: 30px;
    height: 30px;
  }
</style>
