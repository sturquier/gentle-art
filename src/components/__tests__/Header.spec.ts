import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import Header from '../Header.vue'

describe('Header', () => {
  it('renders properly', () => {
    const wrapper = mount(Header)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
