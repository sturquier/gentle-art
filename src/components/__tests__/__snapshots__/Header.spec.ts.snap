// Vitest Snapshot v1

exports[`Header > renders properly 1`] = `
"<v-app-bar>
  <router-link to=\\"/\\"><img alt=\\"logo\\" class=\\"logo\\" src=\\"/src/assets/images/logo.svg\\"></router-link>
</v-app-bar>"
`;
